### PR TITLE
Upgrade: Vert.x 3.8.1 and Axle clients 0.0.9

### PR DESCRIPTION
**Pull request by tsegismont**

### CODE REVIEW

**Comment by tsegismont** [comment]
@cescoffier can you please review? Thanks

**Comment by cescoffier** [comment]
SmallRye Reactive Streams Operators and SmallRye Reactive Messaging have not been released. So, please hold  on this one until it's done.

File: build-parent/pom.xml [hunk 1]
```diff
@@ -29,8 +29,8 @@
            what we work with by self downloading it: -->
         <graal-sdk.version-for-documentation>19.2.0</graal-sdk.version-for-documentation>
         <rest-assured.version>3.3.0</rest-assured.version>
-        <axle-client.version>0.0.7</axle-client.version>
```
[the remark this answers is not in the document]
Comment (tsegismont) [reply build-parent/pom.xml]: Importing boms does not import properties, unfortunately...

**Comment by tsegismont** [comment]
@cescoffier why does the RS Operators and Messaging can't be handled in another PR?

**Pull request closed without merging**